lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -218,5 +218,6 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "transcripts": transcripts,
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
+        "only_on_web" : video_descriptor.unavailable_on_mobile,
```

Comment:
if not mobile_available, the following fields should be set to `None` (or scrubbed depending on what older clients can handle): video_url, video_thumbnail_url, transcripts.